Bugzilla/Install/DB.pm

```diff
@@ -1933,12 +1931,11 @@ sub _convert_groups_system_from_groupset {
 
     # Identify admin group.
     my ($admin_gid)
-      = $dbh->selectrow_array("SELECT id FROM groups WHERE name = 'admin'");
+      = $dbh->selectrow_array("SELECT id FROM `groups` WHERE name = 'admin'");
```

Comment:
Let's do `$dbh->quote_identifier()` here

Reply:
Missed that one.